docs: Add some doc comments to enums (#2285)
slint-ui/slint: docs/language/src/builtins/enums.md

## `ImageFit`

 This enum defines how the source image shall fit into the image element.

> * **`fill`**: Scales and stretches the image to fit the width and height of the element.

Reviewer: ```suggestion
* **`fill`**: Scales and stretches the source image to fit the width and height of the `Image` element.
```

> * **`contain`**: The source image is scaled to fit into the image element's dimension while preserving the aspect ratio.

Reviewer: ```suggestion
* **`contain`**: The source image is scaled to fit into the `Image` element's dimension while preserving the aspect ratio.
```

> * **`cover`**: The source image is scaled to cover into the image element's dimension while preserving the aspect ratio.

Reviewer: ```suggestion
* **`cover`**: The source image is scaled to cover into the `Image` element's dimension while preserving the aspect ratio. If the aspect ratio of the source image doesn't match the element's one, then the image will be clipped to fit.
```

## `LayoutAlignment`

>  Enum representing the alignment property of a BoxLayout or HorizontalLayout.
>  See [alignment](concepts/layouting.md#alignment)

Reviewer: ```suggestion
 Enum representing the [alignment](concepts/layouting.md#alignment) property of a BoxLayout or HorizontalLayout.
```

* **`stretch`**: Use the minimum size of all elements in a layout, distribute remaining space
    based on `*-stretch` among all elements.
* **`center`**: Use minimum size for all elements, distribute remaining space evenly before the

>     first and after the last element

Reviewer: ```suggestion
    first and after the last element.
```

> * **`start`**: Use minimum size for all elements, put remaining space after the last element

Reviewer: ```suggestion
* **`start`**: Use minimum size for all elements, put remaining space after the last element.
```

Reviewer: Are you sure that it's the minimum size that's used? I thought it's the preferred size. @ogoffart  ?

Same goes for center.

AFAICS it's the preferred size that's used: [testcase](https://slint-ui.com/releases/0.3.5/editor/?snippet=import+%7B+VerticalBox+%7D+from+%22std-widgets.slint%22%3B%0Aexport+component+Demo+%7B%0A++++VerticalBox+%7B%0A++++++++alignment%3A+center%3B%0A++++++++Rectangle+%7B%0A++++++++++++background%3A+green%3B%0A++++++++++++min-height%3A+20px%3B%0A++++++++++++preferred-height%3A+100px%3B%0A++++++++%7D%0A++++++++Rectangle+%7B%0A++++++++++++background%3A+red%3B%0A++++++++++++min-height%3A+30px%3B%0A++++++++%7D%0A++++%7D%0A%7D%0A)

* **`end`**: Use minimum size for all elements, put remaining space before the first

>     element

Reviewer: ```suggestion
    element.
```

## `PointerEventKind`

 The enum reports what happened to the `PointerEventButton` in the event

> * **`cancel`**: The action was cancelled
> * **`down`**: The button was pressed
> * **`up`**: The button was released

Reviewer: ```suggestion
* **`cancel`**: The action was cancelled.
* **`down`**: The button was pressed.
* **`up`**: The button was released.
```

## `StandardButtonKind`

>  This enum is used to add standard buttons to a `Dialog`. The look and positioning
>  of these `StandardButton`s depends on the environment (OS, UI environment, etc.)
>  the application runs in.

Reviewer: ```suggestion
 Use this enum to add standard buttons to a `Dialog`. The look and positioning
 of these `StandardButton`s depends on the environment (OS, UI environment, etc.)
 the application runs in.
```

> * **`ok`**: A "OK" button that accepts a `Dialog`, closing it

Reviewer: ```suggestion
* **`ok`**: A "OK" button that accepts a `Dialog`, closing it when clicked.
```

> * **`cancel`**: A "Cancel" button that rejects a `Dialog`, closing it

Reviewer: ```suggestion
* **`cancel`**: A "Cancel" button that rejects a `Dialog`, closing it when clicked.
```

* **`apply`**: A "Apply" button that should accept values from a `Dialog` without closing it.

> * **`close`**: A "Close" button which should close a `Dialog` without looking at values

Reviewer: ```suggestion
* **`close`**: A "Close" button, which should close a `Dialog` without looking at values.
```

> * **`reset`**: A "Reset" button which should reset the `Dialog` to its start state

Reviewer: ```suggestion
* **`reset`**: A "Reset" button, which should reset the `Dialog` to its initial state.
```

> * **`help`**: A "Help" button

Reviewer: ```suggestion
* **`help`**: A "Help" button, which should bring up context related documentation when clicked.
```

> * **`yes`**: A "Yes" button, used to confirm an action

Reviewer: ```suggestion
* **`yes`**: A "Yes" button, used to confirm an action.
```

> * **`no`**: A "No" button, used to negate an action

Reviewer: Deny is the opposite of  confirm :)

```suggestion
* **`no`**: A "No" button, used to deny an action.
```

* **`abort`**: A "Abort" button, used to abort an action

> * **`retry`**: A "Retry" button, used to retry a failed action

Reviewer: ```suggestion
* **`retry`**: A "Retry" button, used to retry a failed action.
```

> * **`ignore`**: A "Ignore" button, used to ignore a failed action

Reviewer: ```suggestion
* **`ignore`**: A "Ignore" button, used to ignore a failed action.
```
